api/v1alpha3/azureimage_validation.go

```diff
@@ -44,6 +44,18 @@ func ValidateImage(image *Image, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+// ValidateSSHKey validates an SSHKey
+func ValidateSSHKey(sshKey string, fldPath *field.Path) field.ErrorList {
```

Comment:
this doesn't belong in azureimage_validation.go, it should be in its own file azuremachine_validation